app/institutions/dashboard/-components/institutional-users-list/styles.scss

```diff
@@ -247,3 +252,38 @@ input:checked + .slider::before {
 .slider.round::before {
     border-radius: 50%;
 }
+
+.total-users {
+    margin-right: auto; /* Aligns text to the left */
+    display: flex; /* Allows for consistent alignment with other elements */
+    align-items: center;
+}
+
+.total-users label {
+    font-size: 18px;
+    margin-bottom: 0;
+    font-weight: normal;
+}
+
+.total-users-count {
+    font-size: 24px;
+    margin-right: 5px;
+    font-weight: bold;
+}
+
+/* Add media query for mobile devices */
+@media (max-width: 1000px) {
```

Comment:
We've tried to move away from hard-coding pixel values into our CSS. Instead, in the template you could use our `is-mobile` helper
```
<div local-class=table-wrapper {{if (is-mobile 'mobile'}}>
```
then change this line to be `.mobile` and adjust the nested selectors as needed

Reply:
Actually, I think this scrolling behavior and no-wrap on the text is something we may want regardless of mobile or not. Let me doublecheck with product though